custom_components/smart_envi/config_flow.py

```
@@ -0,0 +1,676 @@
+                enabled = parts[2].strip().lower() == "true" if len(parts) > 2 else True
+            except (ValueError, IndexError):
+                errors["time_entries"] = f"Invalid format in entry '{entry_str}'. Use: HH:MM:SS,temp,enabled"
+                continue
+            
+            # Validate time format
+            try:
+                time_parts = time_str.split(":")
+                if len(time_parts) == 2:
+                    time_str = f"{time_str}:00"
+                elif len(time_parts) != 3:
+                    raise ValueError("Invalid time format")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)